website/www/site/content/en/contribute/_index.md

```diff
@@ -73,20 +73,33 @@ detail.
 
 ### Prerequisites
 
- - a GitHub account
- - a Linux, macOS, or Microsoft Windows development environment with Java JDK 8 installed
- - [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
-   changes locally
- - [Go](https://golang.org) 1.12 or later installed for Go SDK development
+ - A GitHub account.
```

Comment:
We should have this page on the it to the Wiki IMO.
And consolidate the overlap in Developer Guides tree
https://cwiki.apache.org/confluence/display/BEAM/Developer+Guides

Java Tips, Python Tips, etc have setup instructions.
I recommend making a page on the wiki with setup instructions that setup everything (Java, Python, etc.)
Link to the other wiki pages, or port the contents to this main setup page.

Also, the wiki is easier to update when you have a suggestion (which I do pretty often).

Reply:
First time contributors will probably come here first and I like having the basic/main instructions for contributors on the website. It's nicer to look at than the wiki with all the branding. We should definitely deduplicate the stuff between this and the wiki (removing stuff from the wiki) to avoid them drifting, probably as a separate task.

Having source control for this guide is a plus since we'll be forced to curate the info more carefully through explicit reviews.


We could also add a link here to the wiki for additional tips and troubleshooting topics.

Reply:
BTW, the link to the wiki is in the **Contribution FAQ** section.